src/App.js

```diff
@@ -58,6 +59,7 @@ const App = () => {
           <AppRoute exact path="/roadmap" component={RoadMap} layout={LayoutDefault} />
           <AppRoute exact path="/airdrop" component={AirDrop} layout={LayoutDefaultSlim} />
           <AppRoute exact path="/paper-gardens" component={Chapters} layout={LayoutDefaultSlim} />
+          <AppRoute exact path="/About" component={About} layout={LayoutDefault} />
```

Comment:
could we change the path to all lowercase?

Reply:
path changed to lowercase